documentation/library_guide/parallel_api_main.rst

```diff
@@ -28,13 +28,10 @@ that may be passed to algorithms such as reduce or transform.
 .. toctree::
    :maxdepth: 2
    :titlesonly:
-   :glob:
    :hidden:
 
    parallel_api/execution_policies
-   parallel_api/iterators
    parallel_api/parallel_range_algorithms
    parallel_api/additional_algorithms
    parallel_api/pass_data_algorithms
-   parallel_api/async_api
-   parallel_api/range_based_api
```

Comment:
With range-based APIs moving into the oneapi::dpl namespace now shouldn't the range-based API be listed here instead of in the Technology Preview section?

Reply:
After more thinking, I prefer to move it back. It does not seem to need more attention at this stage.